reference_implementations/aws/offline/06_redshift.tf

```diff
@@ -0,0 +1,94 @@
+# Create a VPC with a Private Subnet
+resource "aws_vpc" "my_vpc" {
```

Comment:
Change the resource name to something else, `my_*` kind of names makes it look a bit unpolished.